wrangler-proto/src/main/java/co/cask/wrangler/proto/db/AllowedDriverInfo.java

```diff
@@ -29,12 +29,14 @@
   private final String name;
   @SerializedName("default.port")
   private final String port;
+  private final boolean basicAllowed;
 
-  public AllowedDriverInfo(String clazz, String label, String tag, String name, String port) {
+  public AllowedDriverInfo(String clazz, String label, String tag, String name, String port, boolean basicAllowed) {
```

Comment:
so we know in future, can you add a comment as to why we maintain two classes for what seem like similar things?

Reply:
I'm quite unsure why there are two classes for this